scripts/__input_class_cursor/__input_class_cursor.gml

```diff
@@ -101,6 +102,72 @@ function __input_class_cursor() constructor
         //Don't update the cursor if the mouse recently moved or we're rebinding controls
         if (global.__input_cursor_verbs_valid && (!global.__input_pointer_moved || !_can_use_mouse) && (__player.__rebind_state <= 0))
         {
+            //Gyro
+            var _motion = undefined;            
+            var _gamepad_index = __player.__source_get_gamepad();
```

Comment:
How will this behave in multidevice mode?

Reply:
First only. Followed an existing pattern (from where I don’t immediately recall). I can use all instead for mixed & multi by summing, since it works surprisingly well for gyro, same as two mice lugged in, basically